src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/RazorConfiguration.cs

```diff
@@ -11,22 +11,22 @@ public sealed record class RazorConfiguration(
     RazorLanguageVersion LanguageVersion,
     string ConfigurationName,
     ImmutableArray<RazorExtension> Extensions,
-    bool UseConsolidatedMvcViews = false,
-    bool ForceRuntimeCodeGeneration = false)
+    RazorLanguageFeatureFlags RazorLanguageFeatureFlags,
```

Comment:
I can't think of a better name, but as mentioned elsewhere a comment about these being owned by the language server, and not serialized, would be good. `LanguageServerFlags` maybe?